requirements.txt

```diff
@@ -16,5 +16,5 @@ MAVProxy==1.8.5
 monotonic==1.5
 pyserial==3.4
 urllib3==1.24.2
-opencv-python==4.1.1
-zlib==1.0
+opencv-python==4.1.1.*
```

Comment:
is this star * supposed to be here?

Reply:
It is. Running cv2.__version__ shows that we're using opencv-python 4.1.1 but it doesn't correspond with any pip module. There are a bunch of releases under 4.1.1, it's probably a better idea to just choose a specific one I guess.

Reply:
Gotcha, I am not too concerned with which minor version we are dealing with. Usually the serious changes come with the major releases so that is fine how it is